scripts/gha/tests

```diff
@@ -105,6 +105,15 @@ echo "Waiting for K3s kubeconfig to be generated"
   --exit-command "dump_rancher_logs" `# Dump logs to find out why K3s did not start` \
   "stat /etc/rancher/k3s/k3s.yaml &>/dev/null"
 
+echo "Waiting up to 5 minutes for the api-extension deployment"
```

Comment:
The same change should be done in `scripts/provisioning-tests` and `scripts/test`